pkg/registry/apis/ofrep/register.go

```diff
@@ -240,50 +245,64 @@ func (b *APIBuilder) GetAPIRoutes(gv schema.GroupVersion) *builder.APIRoutes {
 }
 
 func (b *APIBuilder) oneFlagHandler(w http.ResponseWriter, r *http.Request) {
+	ctx, span := tracer.Start(r.Context(), "ofrep.handler.evalFlag")
+	defer span.End()
+
 	if !b.validateNamespace(r) {
 		b.logger.Error(namespaceMismatchMsg)
+		span.RecordError(errors.New(namespaceMismatchMsg))
```

Comment:
btw, `tracing.Error` and `tracing.Errorf` can be used instead of `span.RecordError`, and as a bonus the span status will be updated to error

Reply:
Thank you! Updated the PR